transformers/storage/median/initializers/median_knc/initializer.go

```diff
@@ -0,0 +1,8 @@
+package median_knc
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers"
+)
+
+var StorageTransformerInitializer = initializers.GenerateStorageTransformerInitializer(constants.GetContractAddress("MEDIAN_WBTC"))
```

Comment:
Should this be getting the `MEDIAN_KNC` address instead of `MEDIAN_WBTC`?

Reply:
yes, nice catch!